src/Util_TEST.cc

```diff
@@ -172,6 +172,100 @@ TEST(Util_TEST, emptyENV)
   EXPECT_TRUE(var.empty());
 }
 
+/////////////////////////////////////////////////
+TEST(Util_TEST, env_set)
```

Comment:
```suggestion
TEST(Util_TEST, envSet)
```

Same for other tests below

Reply:
d7821e8